README.md

```diff
@@ -16,51 +16,101 @@ If you need more help, please contact us at [community@oscafrica.org](mailto:com
 
 Want to know more about Open Source Community Africa? Read our **Introduction** section:
 
-{% page-ref page="about/introduction.md" %}
```

Comment:
This is an intentional behavior. This repository is a copy of our Gitbook integration hence the use of handlebars to link routes in the production [documentation website](https://docs.oscafrica.org). Changing the automatically generated links flow will break the routing.